VMT/VMT.md

```diff
@@ -191,15 +191,15 @@ Issues that take too much time to be fixed (e.g., more than 2 weeks) or Issues t
 
 Whenever such a case occurs, the Kata Containers [Architecture Committee](https://github.com/kata-containers/community#architecture-committee) is subscribed to the Launchpad bug in order to discuss whether or not it's imperative to keep that particular bug private.
 
-The same process is followed if an embargoed issue is, accidentally or not, leaked to the public. In all likelyhood that Issue will then be moved to the Public state.
+The same process is followed if an embargoed issue is, accidentally or not, leaked to the public. In all likelihood that Issue will then be moved to the Public state.
```

Comment:
wow, well, the Internet (which can never be wrong...) says this is a correct fix - but it still looks so wrong to my eyes! 👀

Reply:
It does indeed look almost like an Americanism to mine eyes too ;)